test/unit/server-configmap.bats

```diff
@@ -92,23 +98,70 @@ load _helpers
   [ "${actual}" = "bar" ]
 }
 
-@test "server/ConfigMap: standalone extraConfig is set" {
+@test "server/ConfigMap: standalone extraConfig is set as JSON" {
   cd `chart_dir`
-  local actual=$(helm template \
+  local data
+  data=$(helm template \
       --show-only templates/server-config-configmap.yaml  \
       --set 'server.standalone.enabled=true' \
-      --set 'server.standalone.config="{\"hello\": \"world\"}"' \
+      --set 'server.standalone.config=\{\"hello\": \"world\"\}' \
       . | tee /dev/stderr |
-      yq '.data["extraconfig-from-values.hcl"] | match("world") | length' | tee /dev/stderr)
-  [ ! -z "${actual}" ]
+      yq '.data')
+  [ "$(echo "${data}" | \
+    yq '(. | length) == 1')" = "true" ]
+  [ "$(echo "${data}" | \
+    yq '."extraconfig-from-values.hcl" == "{\"disable_mlock\":true,\"hello\":\"world\"}"')" = 'true' ]
```

Comment:
I think we could reduce some of the escapes here with `yq -r` (and I had to use `yq -Y` on .data for some reason):

```suggestion
      yq -Y '.data')
  [ "$(echo "${data}" | \
    yq '(. | length) == 1')" = "true" ]
  local extraConfig=$(echo "${data}" | yq -r '."extraconfig-from-values.hcl"')
  [ "${extraConfig}" == '{"disable_mlock":true,"hello":"world"}' ]
```

Reply:
Agreed, but I think I prefer letting yq do the value evaluation for us.